classes/user/Repository.php

```diff
@@ -446,4 +450,162 @@ public function getAdminUsers(): LazyCollection
     {
         return $this->dao->getAdminUsers();
     }
+
+    /**
+     * build a permission map for a manager over a set of users
+     */
+    public function permissionMapForManager(int $managerUserId, array $userIds): array
```

Comment:
@Hafsa-Naeem, I don't think some of these functions are used now that the PR has been simplified.